Add um::ole2::OleInitialize, shared::wtypes::DVASPECT

ArturKovacs opened the pull request:
These were missing in order to implement an IDropTarget that handles file drop.

ArturKovacs committed:
Add um::ole2::OleInitialize, shared::wtypes::tagDVASPECT

This adds functionality required for IDropTarget.

ArturKovacs committed:
Merge branch '0.3' of https://github.com/retep998/winapi-rs into 0.3

Merging due to implementation of IDragDrop functionality namely:
um::ole2::OleInitialize
shared::wtypes::tagDVASPECT

retep998 commented on src/shared/wtypes.rs:
```diff
@@ -62,6 +62,12 @@ ENUM!{enum VARENUM {
     VT_RESERVED = 0x8000,
     VT_ILLEGAL = 0xffff,
 }}
+ENUM!{enum tagDVASPECT {
```
This enum should be named `DVASPECT`.

ArturKovacs committed:
Fixed previous commit

Renamed tagDVASPECT to DVASPECT.

Chaned use order in um::ole2

Added "minwindef" dependecy for ole2 to the build script.

ArturKovacs commented:
Sorry for the 'use' ordering issues. My attention is all over the place.

Is there a good way I can check these rules locally?

ArturKovacs committed:
Fix for use ordering.

retep998 commented:
`cargo test --all-features` will run all the tests if you want to test locally.

ArturKovacs commented:
Thank you next time I'll be wiser and run the tests locally before committing.

Is there anything that prevents this to be merged?

ArturKovacs commented:
Hello again!
I'm looking forward to seeing this merged.
If there's any reason this can't be merged in this current state I would be happy to follow your instructions.
